etc/sysconfig/pacemaker.in

```diff
@@ -263,10 +254,24 @@
 # in PEM format. This file must be readble by Pacemaker daemons (that is, it
 # must allow read permissions to either the @CRM_DAEMON_USER@ user or the
 # @CRM_DAEMON_GROUP@ group. If set, along with PCMK_ca_file and PCMK_cert_file,
-# X509 authentication will be enabled for remote CIB connections.
+# X509 authentication will be enabled for Pacemaker Remote and remote CIB
+# connections.
 #
 # Default: PCMK_key_file=""
 
+# PCMK_authkey_location
+#
+# As an alternative to using X509 authentication for Pacemaker Remote
+# connections, use the contents of this file as the authorization key. This
+# file must be readable by Pacemaker daemons (that is, it must allow read
+# permissions to either the @CRM_DAEMON_USER@ user or the @CRM_DAEMON_GROUP@
+# group), and its contents must be identical on all nodes.
+#
+# This is an older authorization mechanism and X509 certificates are now
+# preferred.
```

Comment:
I don't think we need to prefer them, shared key is fine for most users. It's no less secure, the difference is mainly about key management.